### PR TITLE
Add namespaces specific to collision implementation

### PR DESCRIPTION
**Pull request by Levi-Armstrong**

### CODE REVIEW

**Comment by Levi-Armstrong** [comment]
@Jmeyer1292 Friendly reminder.

File: tesseract_collision/include/tesseract_collision/bullet/bullet_utils.h [hunk 1]
```diff
@@ -107,11 +109,13 @@ class CollisionObjectWrapper : public btCollisionObject
...
-    return m_name == other.m_name && m_type_id == other.m_type_id &&
-           m_shapes.size() == other.m_shapes.size() &&
+    return m_name == other.m_name && m_type_id == other.m_type_id && m_shapes.size() == other.m_shapes.size() &&
            m_shape_poses.size() == other.m_shape_poses.size() &&
            std::equal(m_shapes.begin(), m_shapes.end(), other.m_shapes.begin()) &&
-           std::equal(m_shape_poses.begin(), m_shape_poses.end(), other.m_shape_poses.begin(), [&](const Eigen::Isometry3d& t1, const Eigen::Isometry3d& t2) { return t1.isApprox(t2); });
+           std::equal(m_shape_poses.begin(),
+                      m_shape_poses.end(),
+                      other.m_shape_poses.begin(),
+                      [&](const Eigen::Isometry3d& t1, const Eigen::Isometry3d& t2) { return t1.isApprox(t2); });
```
Comment (Jmeyer1292) [inline tesseract_collision/include/tesseract_collision/bullet/bullet_utils.h]: Why do you capture all values-by reference (the ```[&]```) into your lambda-closure?

**Comment by Levi-Armstrong** [comment]
Oh, I did not notice I committed that directory. It has been removed.

**Comment by Levi-Armstrong** [comment]
@Jmeyer1292 I made the changes to the lambda functions and changed the lambda's that were taking `std::string link` to `const std::string& link`.

**Pull request merged**